keylime/crypto.py

```diff
@@ -78,21 +80,22 @@ def rsa_sign(key, message):
         padding.PSS(
             mgf=padding.MGF1(hashes.SHA256()),
             salt_length=padding.PSS.MAX_LENGTH
-            ),
-            hashes.SHA256()
-        )
-    return signature
+        ),
+        hashes.SHA256()
+    )
+    return base64.b64encode(signature)
```

Comment:
Note: My IDE is automatically fixing the lint of python files (using pep8). The main change for this PR is the line should here on line 86